File: CS 1450 Object Oriented Programming/Finals/Code/finals_1.py
number = int(input("Choose a number between 0 - 50: "))
factorial = 1
if number < 0:
	print("Cannot perform the calculation, the number you entered was negative!")
elif number > 50:
	print("Cannot perform the calculation, the number you entered was bigger than 50!")
else:
	for i in range(1, number + 1):
		factorial = factorial * i
	print("The factorial of", number, "is:", factorial)

File: CS 3140 The C++ Language/Finals/Code/finals_1.cpp
#include <iostream>
using namespace std;

int factorial(int a)
{
	int factorial=1;
	for (int i=1; i<=a; i++)
	{
		factorial = factorial * i;
	}
	return factorial;
}


int main()
{
	int number;

	cout << "Please input a number between 0 and 50:" << endl;
	cin >> number;
	if (number < 0)
	{
		cout << "Number was negative, can't perform calculation." << endl;
	}
	else if (number > 10)
	{
		cout << "Number was bigger than 10, can't perform calculation." << endl;
	}
	else
	{
		cout << "The factorial of your number " << number <<" is: " << factorial(number) << endl;
	}

	return 0;
}
